apps/site/lib/site_web/templates/schedule/_line.html.eex

```diff
@@ -62,11 +62,7 @@
 
     <div class="col-md-5 col-lg-4 col-lg-offset-1 m-schedule-page__schedule-finder-or-note">
       <%= unless Enum.empty?(@branches) do %>
-        <div id="react-schedule-note-root">
-          <%= if @schedule_page_data[:schedule_note] do %>
-            <%= Site.React.render("ScheduleNote", %{scheduleNote: @schedule_page_data.schedule_note, className: "m-schedule-page__schedule-notes--desktop"}) %>
-          <% end %>
-        </div>
+        <div id="react-schedule-note-root"></div>
```

Comment:
I forgot there's one more place where the deleted `ScheduleNote` component is called, and it's mentioned in the error message in the failing Docker build -- check `react_app.js`!